static/bidder-params/adocean.json

```diff
@@ -6,9 +6,14 @@
   "properties": {
     "emiter": {
       "type": "string",
-      "description": "AdOcean emiter",
+      "description": "Deprecated, use emitterPrefix instead. AdOcean emiter",
```

Comment:
Please update [dev-docs ](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/adocean.md)to reflect changes in the parameters